pkg/util/config/config.go

```diff
@@ -163,13 +163,13 @@ func Copy(c *Config) Config {
 	return cfg
 }
 
-func validate(cfg *Config) (err error) {
+func Validate(cfg *Config) error {
```

Comment:
Why to capitalize this and `ProcessField`? They're in the same package with the tests, so as far as I remember should be visible anyway.